addons/sales_team/views/crm_team_views.xml

```diff
@@ -125,28 +125,22 @@
         <field name="model">crm.team</field>
         <field name="priority">10</field>
         <field name="arch" type="xml">
-            <kanban class="o_kanban_dashboard o_crm_team_kanban" create="0" can_open="0" sample="1">
-                <field name="name"/>
-                <field name="user_id"/>
-                <field name="member_ids"/>
-                <field name="color"/>
-                <field name="currency_id"/>
-                <field name="is_favorite"/>
+            <kanban class="o_crm_team_kanban" highlight_color="color" create="0" can_open="0" sample="1">
                 <templates>
                     <t t-name="kanban-menu">
                         <div class="container">
                             <div class="row">
-                                <div class="col-4 o_kanban_card_manage_section o_kanban_manage_view">
+                                <div class="col-4 o_kanban_manage_view">
```

Comment:
remove o_kanban_manage_view ? I couldn't find any associated scss

Reply:
Hello

Through the classes divs are getting targetted in the inheriting views. https://github.com/odoo/odoo/pull/180140/files#diff-7bf27f0e1c7f9111036f3330c5e0c25d71d39ed6dbe6825ceb74e9c4c1cb526dL124

if you recommend then I can give names to divs instead of class so that the divs can be targetted

Reply:
Indeed, we're using names rather than classnames for xpath purposes